src/core/fs.h

```diff
@@ -471,8 +487,12 @@ fs_createtime(
 #else
   struct stat statbuf = {0};
   stat(filename, &statbuf);
+#ifdef __APPLE__
```

Comment:
i think this is obsolete now, the ifdef is in the else path of another ifdef __APPLE__, no?